### PR TITLE
DRAFT: send messages to mobile account

### PR DESCRIPTION
**Pull request by DavidLawes**
This is in draft at the moment, I don't think we need to deploy this to PROD right now. At the moment MAPI only creates the required resources for CODE (see here for ref: https://github.com/guardian/mobile-apps-api/pull/2703)

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: src/main/scala/com/gu/mobile/content/notifications/ContentLambda.scala [hunk 1]
```diff
@@ -7,8 +7,13 @@ object ContentLambda extends Lambda {
 
   def processContent(content: Content): Boolean = {
     logger.info(s"Processing ContendId: ${content.id} Published at: ${content.getLoggablePublicationDate}")
+
+    // Temporary: this is to support the mobile text-to-speech experiment
+    sqs.sendMessage(content.id)
```
Comment (DavidLawes) [inline src/main/scala/com/gu/mobile/content/notifications/ContentLambda.scala:12]: Wrap this to catch/suppress any errors?

**Review by tkgnm — APPROVED** [review]
LGTM!

File: src/main/scala/com/gu/mobile/content/notifications/ContentLambda.scala (hunk 1 above)
Comment (tkgnm) [reply src/main/scala/com/gu/mobile/content/notifications/ContentLambda.scala:12]: Would it make sense to check there is not a message on the SQS with the same `content.id` before pushing onto the queue? Could that happen?

Comment (DavidLawes) [reply src/main/scala/com/gu/mobile/content/notifications/ContentLambda.scala:12]: I think that's a good point - I wondered if we might get messages with the same content id if an article was updated (after first being published). In this case we may want to trigger the audio-generator lambda to make sure the audio includes the latest content. The other kind of thing we were trying to do here is make any temporary extension to this lambda as small as possible, then handle any ugliness/complexity in the audio-generator lambda. 

As part of the of the experiment we also made a note that we might like to send messages (e.g. to our google space) when an audio conversion task was kicked off, so maybe we'd be able to see if this kind of behaviour (multiple subsequent content id appearing on the queue in rapid succession) happens when we've deployed this to the wild.

**Pull request closed without merging**